extras/userdoc/md/documentation/part-1-neurons-and-simple-neural-networks.md

```diff
@@ -17,26 +17,38 @@ discusses in this primer, please visit [nest-initiative.org/](http://nest-initia
 or have a look at the source directory of your NEST installation in the
 subdirectory:
 `pynest/examples/`. For the internals of the NEST simulator you may refer
-to the [publications](publications.md).
+to the [\[1\]](#1),[\[2\]](#2).
 
 ## PyNEST - an interface to the NEST simulator
 
-![Python Interface](../../img/python_interface.png) The Python interpreter
-imports NEST as a module and dynamically loads the NEST simulator kernel
-(`pynestkernel.so`). The core functionality is defined in `hl_api.py`. A
-simulation script of the user (`mysimulation.py`) uses functions defined in this
-high-level API. These functions generate code in SLI (Simulation Language
-Interpreter), the native language of the interpreter of NEST. This interpreter,
-in turn, controls the NEST simulation kernel.
+<table><tr>
+<td style="max-width: 400px;">
+<img src="../../img/python_interface.png" alt="Python Interface" title="Python 
+Interface"/>
+</td>
+<td></td>
+</tr></table>
+
+Figure 1<a id="figure-1"></a>: The Python interpreter imports NEST as a 
+module 
+and 
+dynamically 
+loads 
+the NEST simulator kernel (`pynestkernel.so`). The core functionality is defined 
+in `hl_api.py`. A simulation script of the user (`mysimulation.py`) uses 
+functions defined in this high-level API. These functions generate code in 
+SLI (Simulation Language Interpreter), the native language of the interpreter
+ of NEST. This interpreter, in turn, controls the NEST simulation kernel.
 
 The NEural Simulation Tool (NEST: [www.nest-initiative.org](part-1-neurons-and-simple-neural-networks.md))
 is designed for the simulation of large heterogeneous networks of point neurons.
 It is open source software released under the GPL licence. The simulator comes
-with an interface to Python . illustrates the interaction between the user’s
-simulation script (`mysimulation.py`) and the NEST simulator. contains a
-technically detailed description of the implementation of this interface and
-parts of this text are based on this reference. The simulation kernel is written
-in$$ C++$$ to obtain highest possible performance for the simulation.
+with an interface to Python [\[5\]](#5). [Fig. 1](#figure-1) illustrates the 
+interaction between the user’s simulation script (`mysimulation.py`) and the 
+NEST simulator. [\[2]\](#2) contains a technically detailed description of the 
```

Comment:
Yes, that's what I meant, it's just that your previous message was disturbing because GitHub automatically converted it to a link with only one bracket showing